recipes/cpr/all/conanfile.py

```diff
@@ -82,24 +84,24 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        SSL_CURL_LIBS = {
+            "openssl": "openssl",
+            "darwinssl": "darwinssl",
+            "winssl": "schannel"
```

Comment:
Would it be easier if we used the name value?

Reply:
Sorry, not sure what you're referring to here. Could you clarify?

Reply:
There's a bit of complexity from the CPR values being different from the curl recipe... 

But looking at it again, it's just combinatorics hell, since the options and support have changed a lot it's very complicated